ansible-scylla-node/tasks/common.yml

```diff
@@ -390,14 +390,37 @@
           {% if (_scylla_yaml_map.audit is defined and _scylla_yaml_map.audit == 'table') %}True{% else %}False{% endif %}
   run_once: true
 
+- name: Check if all nodes are up
+  block:
+    - name: Use Scylla API to see if the nodes are up
+      uri:
+        url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/failure_detector/endpoints/"
+        follow_redirects: none
+        method: GET
+      register: _failure_detector_result
+      until: _failure_detector_result.status == 200
+      retries: 3
+      delay: 1
+      ignore_errors: true
+      delegate_to: "{{ item }}"
+      loop: "{{ groups['scylla'] }}"
+      run_once: true
+
+    - name: Set all_nodes_up as a fact
+      set_fact:
+        all_nodes_up: "{% if item.status is defined and item.status == 200 %}{{ True }}{% else %}{{ False }}{% endif %}"
+      when: all_nodes_up is not defined or all_nodes_up|bool
+      loop: "{{ _failure_detector_result.results }}"
+      run_once: true
+
 - name: Adjust replication for system_auth keyspace
   include_tasks: adjust_keyspace_replication.yml
   vars:
     _keyspace: "system_auth"
     _keyspace_replication_strategy: "{{ system_auth_replication_strategy }}"
     _keyspace_rf: "{{ system_auth_rf }}"
   when:
-    - start_scylla_service|bool
+    - all_nodes_up | bool
```

Comment:
Inconsistent amount of whitespaces: let's continue NOT having them in explicit type casting like `some_variable|some_type`.

Reply:
Fixed